src/weatherreport/doc/overview.edoc

```diff
@@ -0,0 +1,23 @@
+@author Basho Technologies, Inc.
```

Comment:
I'm in favour of just collapsing this into the top level `README` and dropping the entire `doc` directory.

Reply:
Yeah, that .edoc file was obsolete. I've deleted it.

Reply:
Dropped.